### PR TITLE
Groups endpoint

### PR DESCRIPTION
**Pull request by RomanMukhin**

### CODE REVIEW

File: app/models/vertigo/rtm/group.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
   module Rtm
-    class Group < Conversation
+    class Group < Vertigo::Rtm::Conversation
+      validates :name, uniqueness: true
+      before_validation :set_name
+
+      private
+
+      def set_name
+        self[:name] = (members.map(&:id) | [creator_id]).sort.join('-')
```
Comment (GeorgeZhukov) [inline app/models/vertigo/rtm/group.rb]: better to use ids or pluck instead map. 
better to replace sort with order.

Comment (RomanMukhin) [inline app/models/vertigo/rtm/group.rb]: pluck and order methods go to the database and the group is not persisted in :before_validation callback on creation.

**Comment by chubchenko** [comment]
+

**Comment by kirillshevch** [comment]
reviewed

**Comment by vshmyhlo** [comment]
:+1:

**Pull request merged**